client/extensions/woocommerce/app/product-categories/form.js

```diff
@@ -218,6 +262,29 @@ class ProductCategoryForm extends Component {
 							</FormFieldSet>
 						</div>
 					</div>
+					<FormFieldSet>
+						<FormLabel>
+							<FormCheckbox checked={ this.state.isTopLevel } onChange={ this.onTopLevelChange } />
+							<span>{ translate( 'Top level category' ) }</span>
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Top level category', { context: 'Categories: New category being created is top level i.e. has no parent'} )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).